src/tensors/cpu/wasm_intgemm_interface.h

```diff
@@ -0,0 +1,159 @@
+#pragma once
+
+/* Main interface for integer matrix multiplication (C = A * B + Bias) for wasm.
+ *
+ * A is typically activations whose rows should be a multiple of 1 (i.e. no restriction) and
+ * columns should be a multiple of 64.
+ * 
+ * B is typically fixed model parameters whose rows should be a multiple of 64 and columns
+ * should be a multiple of 8.
+ * 
+ * All matrices A, B and C are in row-major format.
+ */
+
+#include <cstdint>
+
+using Index = unsigned int;
+
+/**
+ * Prepare B for the Matrix Multiply routine.
+ *
+ * B is prepared in a CPU-dependent format by performing quantization on floating values.
+ *
+ * @param[in]   B_input     An array representing the input 2-D matrix in row-major format.
+ *                          Size of the array = `B_rows` * `B_cols`
+ * @param[in]   scale       The scaling factor (for quantization)
+ * @param[in]   zero_point  The zero point (for quantization)
+ * @param[in]   B_rows      No. of rows of input matrix. It should be a multiple of 64.
+ * @param[in]   B_cols      No. of columns of input matrix. It should be a multiple of 8.
+ * @param[out]  output      An array representing the prepared B matrix in row-major format having
+ *                          the same size as of the input matrix i.e. `B_rows` * `B_cols`
+ */
+void Int8PrepareB(const float* B_input,
+                float scale,
+                int8_t zero_point,
+                Index B_rows,
+                Index B_cols,
+                int8_t* output);
```

Comment:
Add an argument here: `bool transpose=false`

Reply:
@XapaJIaMnu Just to make sure that I understood the function of `bool transpose` argument correctly, could you please confirm following for me:

Assuming shape of the final output matrix `output` is `(B_rows, B_cols)`

1. When `transpose=true`  => shape of input matrix `B_input` is expected to be `(B_cols, B_rows)`
2. When `transpose=false` => shape of input matrix `B_input` is expected to be `(B_rows, B_cols)` (i.e same as `output` matrix)

Reply:
Yes, I think this is the correct behaviour.

Our intgemm interface differs, but we should just fix that.

Reply:
Great. Adding the parameter and the documentation around it.

Reply:
Done. Resolving this as I pushed a commit for this.